layers/+emacs/better-defaults/config.el

```diff
@@ -0,0 +1,20 @@
+;;; config.el --- Better Emacs Defaults Layer configuration variables File
+;;
+;; Copyright (c) 2012-2016 Sylvain Benner & Contributors
+;;
+;; Author: Thomas de Beauchene <thomas.de.beauchene@gmail.com>
+;; URL: https://github.com/syl20bnr/spacemacs
+;;
+;; This file is not part of GNU Emacs.
+;;
+;;; License: GPLv3
+
+(defvar better-defaults-mwim-beginning-of-code-first t
+  "when t, first stroke of C-a will move the cursor to the beginning of code.
+When nil, first stroke will go to the beginning of line.
+Subsequent strokes will toggle between beginning of line and beginning of code.")
+
+(defvar better-defaults-mwim-end-of-code-first t
+  "when t, first stroke of C-a will move the cursor to the end of code (before comments).
```

Comment:
Well spoted ! I changed it along with the variable names which were a bit mysterious if you don't know the package mwim.